.github/workflows/lint-go.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.59.1
+          version: v1.60.3
```

Comment:
can we clean up a bit this file - seems like it is throwing errors about: Warning: Unexpected input(s) 'skip-pkg-cache', 'skip-build-cache', valid inputs are ['version', 'install-mode', 'working-directory', 'github-token', 'only-new-issues', 'skip-cache', 'skip-save-cache', 'problem-matchers', 'args', 'cache-invalidation-interval'], probably it is safe to remove these

Reply:
Where/how are you seeing/reproducing those errors ?

Reply:
it is warning that we are using non-existing config options - https://github.com/vechain/thor/actions/runs/10723392301/job/29736573457 just open golangci-lint

Reply:
I think that was a fluke, I've reran that action and it worked ok.